README.md

```diff
@@ -1,39 +1,39 @@
-# Elixir Stellar Base
 ![Build Badge](https://img.shields.io/github/workflow/status/kommitters/stellar_base/StellarBase%20CI/main?style=for-the-badge)
 [![Coverage Status](https://img.shields.io/coveralls/github/kommitters/stellar_base?style=for-the-badge)](https://coveralls.io/github/kommitters/stellar_base)
 [![Version Badge](https://img.shields.io/hexpm/v/stellar_base?style=for-the-badge)](https://hexdocs.pm/stellar_base)
 ![Downloads Badge](https://img.shields.io/hexpm/dt/stellar_base?style=for-the-badge)
 [![License badge](https://img.shields.io/hexpm/l/stellar_base.svg?style=for-the-badge)](https://github.com/kommitters/stellar_base/blob/main/LICENSE.md)
 
-### ⚠️ Warning! This library is under active development. DO NOT use it in production environments.
```

Comment:
It's not necessary this warning. We can use it in prod?

Reply:
It is not really needed if you follow the semantic versioning it's clear that it is not a stable version yet.